docs/features/dynamic-analysis.work.md

```diff
@@ -0,0 +1,14 @@
+Dynamic Analysis
+===============
+
+This feature enables instrumentation of binaries to collect information for dynamic analysis.
+
+--------------------
+
+TODO:
+
+- Generate code to compute MVIDs less often than once per invoked method.
```

Comment:
Can you change the markup to check boxes instead of bullets so we can keep track of where we started, where we are now, and how far we have to go?